packages/ajna-contracts/contracts/ajna/ajna-actions/AjnaProxyActions.sol

```diff
@@ -360,6 +366,7 @@ contract AjnaProxyActions is IAjnaProxyActions {
         _pull(debtToken, debtAmount);
         IERC20(debtToken).safeApprove(address(pool), debtAmount);
         (, , , , , uint256 lupIndex_) = poolInfoUtils.poolPricesInfo(address(pool));
+        uint256 quoteBalanceBefore = IERC20(debtToken).balanceOf(address(this));
```

Comment:
We should also talk in the future about these functions, because they seem to have very similar behaviour with some differences. Perhaps we could refactor them to not duplicate code